benches/vm_benchmark.rs

```diff
@@ -35,17 +36,18 @@ fn interpret_benchmark(c: &mut Criterion) {
 fn asm_benchmark(c: &mut Criterion) {
     c.bench_function("interpret secp256k1_bench via assembly", |b| {
         let buffer = fs::read("benches/data/secp256k1_bench").unwrap().into();
-        let args: Vec<Bytes> = vec!["secp256k1_bench",
-                                      "033f8cf9c4d51a33206a6c1c6b27d2cc5129daa19dbd1fc148d395284f6b26411f",
-                                      "304402203679d909f43f073c7c1dcf8468a485090589079ee834e6eed92fea9b09b06a2402201e46f1075afa18f306715e7db87493e7b7e779569aa13c64ab3d09980b3560a3",
-                                      "foo",
-                                      "bar"].into_iter().map(|a| a.into()).collect();
-
+        let args = [
+            Ok("secp256k1_bench".into()),
+            Ok("033f8cf9c4d51a33206a6c1c6b27d2cc5129daa19dbd1fc148d395284f6b26411f".into()),
+            Ok("304402203679d909f43f073c7c1dcf8468a485090589079ee834e6eed92fea9b09b06a2402201e46f1075afa18f306715e7db87493e7b7e779569aa13c64ab3d09980b3560a3".into()),
+            Ok("foo".into()),
+            Ok("bar".into()),
+        ].into_iter();
```

Comment:
Can we use a `.into_iter().map(|a| Ok(a.into()))` or something like that instead?

Reply:
The code here has been optimized

Reply:
I'm afraid I don't follow here. I mean if we can do something like this:

```
let args = [
            "secp256k1_bench",
            "033f8cf9c4d51a33206a6c1c6b27d2cc5129daa19dbd1fc148d395284f6b26411f",
            "304402203679d909f43f073c7c1dcf8468a485090589079ee834e6eed92fea9b09b06a2402201e46f1075afa18f306715e7db87493e7b7e779569aa13c64ab3d09980b3560a3",
            "foo",
            "bar",
        ].into_iter().map(|a| Ok(a.into()));
```

Reply:
Please recheck the code; I didn't push the commit earlier.